src/main/java/ch/njol/skript/effects/EffLog.java

```diff
@@ -77,42 +77,54 @@ public void close() {
 	private Expression<String> messages;
 	@Nullable
 	private Expression<String> files;
-	
+
+	private Level logLevel = Level.INFO;
+	private static <LogLevel> String getLogPrefix(LogLevel logLevel) {
+		String timestamp = SkriptConfig.formatDate(System.currentTimeMillis());
+		if (logLevel == Level.INFO)
+			return "[" + timestamp + "]";
+		return "[" + timestamp + " " + logLevel + "]";
+	}
+
 	@SuppressWarnings({"unchecked", "null"})
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parser) {
 		messages = (Expression<String>) exprs[0];
 		files = (Expression<String>) exprs[1];
+		if (parser.mark == 1) {
+			logLevel = Level.WARNING;
+		} else if (parser.mark == 2) {
+			logLevel = Level.SEVERE;
+		}
 		return true;
 	}
 	
 	@SuppressWarnings("resource")
 	@Override
-	protected void execute(final Event e) {
-		for (final String message : messages.getArray(e)) {
+	protected void execute(Event event) {
+		for (String message : messages.getArray(event)) {
 			if (files != null) {
-				for (String s : files.getArray(e)) {
-					s = s.toLowerCase(Locale.ENGLISH);
-					if (!s.endsWith(".log"))
-						s += ".log";
-					if (s.equals("server.log")) {
-						SkriptLogger.LOGGER.log(Level.INFO, message);
-						continue;
+				for (String logFile : files.getArray(event)) {
+					logFile = logFile.toLowerCase(Locale.ENGLISH);
+					if (!logFile.endsWith(".log"))
+						logFile += ".log";
+					if (logFile.equals("server.log")) {
+						SkriptLogger.LOGGER.log(logLevel, message);
```

Comment:
the old behavior was to continue after this. is there a reason it was/can be removed?
```suggestion
						SkriptLogger.LOGGER.log(logLevel, message);
						continue;
```